metals/src/main/scala/scala/meta/internal/metals/JavaInteractiveSemanticdb.scala

```diff
@@ -169,63 +169,24 @@ class JavaInteractiveSemanticdb(
       }
     } else Nil
   }
-
-  private def addExportsFlags: List[String] = {
```

Comment:
Could we keep the process way of creating Java semanticdb as a fallback if the flags are not enabled?